octree_container.h

```diff
@@ -64,7 +64,11 @@ namespace OrthoTree
     : m_geometryCollection(geometryCollection.begin(), geometryCollection.end())
     {
       if (isParallelCreation)
+#if !__APPLE__
         OrthoTreeCore::template Create<std::execution::parallel_unsequenced_policy>(m_tree, m_geometryCollection, maxDepthNo, boxSpace, maxElementNoInNode);
+#else
+        { /* Execution policies are not available on Apple platforms yet */ }
```

Comment:
The `else` branch with `OrthoTreeCore::Create` should be executed instead of this. And a run-time assert should be added. (and the same for the followings ofc.)